tests/network-tests/src/flows/content/createApp.ts

```diff
@@ -30,28 +30,7 @@ export async function createApp({ api, query }: FlowProps): Promise<void> {
     () => query.getAppsByName(appOwnedByMember),
     (appsByName) => {
       assert.equal(appsByName?.[0]?.name, appOwnedByMember)
-      assert.equal(appsByName?.[0].ownerMember?.id, member.memberId.toString())
-      assert.equal(appsByName?.[0].isLeadOwned, false)
-      assert.equal(appsByName?.[0]?.category, appMetadata.category)
-      assert.equal(appsByName?.[0]?.oneLiner, appMetadata.oneLiner)
-      assert.equal(appsByName?.[0]?.description, appMetadata.description)
-      assert.equal(appsByName?.[0]?.termsOfService, null)
-      assert.equal(appsByName?.[0]?.websiteUrl, null)
-      assert.deepEqual(appsByName?.[0]?.platforms, appMetadata.platforms)
-    }
-  )
-
-  // app created by lead
-
-  const appOwnedByLead = 'app_owned_by_lead'
-  await api.createApp(appOwnedByLead, appMetadata)
-
-  await query.tryQueryWithTimeout(
-    () => query.getAppsByName(appOwnedByLead),
-    (appsByName) => {
-      assert.equal(appsByName?.[0]?.name, appOwnedByLead)
-      assert.equal(appsByName?.[0].ownerMember?.id, null)
-      assert.equal(appsByName?.[0].isLeadOwned, true)
+      assert.equal(appsByName?.[0].ownerMember.id, member.memberId.toString())
       assert.equal(appsByName?.[0]?.category, appMetadata.category)
       assert.equal(appsByName?.[0]?.oneLiner, appMetadata.oneLiner)
       assert.equal(appsByName?.[0]?.description, appMetadata.description)
```

Comment:
Maybe I'm nitpicking here, but we could rename all the variables like appOwnedByMember to just `app` or `newApp`. Just for the sake of clarity? WDYT? We don't need to differentiate anymore between the app owned by the lead and the app owned by members.